README.md

```diff
@@ -67,7 +69,7 @@ fingers will both grasp what's going on. Let's look at the following example:
 
 Here's a story:
 
-1. You fork the repository `https://github.com/learn-co-students/awesome-lab`
+1. You fork the [fake] repository `https://github.com/learn-co-students/awesome-lab`
```

Comment:
English tends to use `()` versus `[]`:

instead of "fake" which implies value why not:

* mockup
* strawman
* mock
* scarecrow
* placeholder
* demonstration

etc.